sources/api/migration/migrations/v1.1.3/kubelet-cpu-manager-state/src/main.rs

```diff
@@ -0,0 +1,61 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::{error, migrate, Migration, MigrationData, Result};
+use snafu::ResultExt;
+use std::fs;
+use std::io;
+use std::path::Path;
+use std::process;
+
+const CPU_MANAGER_POLICY_CHECKPOINT: &str = "/var/lib/kubelet/cpu_manager_state";
+
+/// forward - We always remove the state file on boot, therefore we don't need to explicitly
+/// remove the file during forward migration.
+/// backward - We remove cpu manager policy checkpoint value on downgrade, since older versions did not
+/// clean up this state file on boot.
+pub struct CpuManagerPolicyCleaner;
+
+impl Migration for CpuManagerPolicyCleaner {
+    fn forward(&mut self, input: MigrationData) -> Result<MigrationData> {
+        println!("CpuManagerPolicyCleaner has no work to do on upgrade.");
+        Ok(input)
+    }
+
+    fn backward(&mut self, input: MigrationData) -> Result<MigrationData> {
+        // removing existing cpu_manager_policy_state file
+        if Path::new(CPU_MANAGER_POLICY_CHECKPOINT).exists() {
```

Comment:
I don't think this check is helping because we already check for NotFound and can indicate that it didn't exist.  I think it'd be better to remove this because then we'd get output from the migration rather than silence.